lib/middlewares.ts

```diff
@@ -13,6 +13,24 @@ interface DashboardTokenPayload extends JwtPayload {
   app: string;
 }
 
+export const withErrorJsonified: Middleware = (handler) => async (request) => {
+  try {
+    return await handler(request);
+  } catch (error) {
+    console.log(error);
+
+    let errorMessage = "Something went wrong!";
+    if (error instanceof Error) {
+      errorMessage = error.message;
+    }
+
+    return Response.InternalServerError({
+      success: false,
+      message: errorMessage,
```

Comment:
Would something like this work?
```suggestion
      message: error instanceof Error ? error.message : "Something went wrong!",
```